src/main/java/org/takes/facets/fallback/FbStatus.java

```diff
@@ -45,7 +47,46 @@
  */
 @EqualsAndHashCode(callSuper = true)
 public final class FbStatus extends FbWrap {
-
+    /**
+     * Ctor.
+     * @param code HTTP status code
+     * @since 0.16.10
+     */
+    public FbStatus(final int code) {
+        this(new Condition<Integer>() {
+            @Override
+            public boolean fits(final Integer status) {
+                return code == status;
```

Comment:
@komaz autoboxing will be used here, but maybe let's make it visible? .`intValue()` or `equals()`